backend/tracim_backend/__init__.py

```diff
@@ -234,6 +236,9 @@ def web(global_config: OrderedDict, **local_settings) -> Router:
     configurator.include(add_cors_support)
     # make sure to add this before other routes to intercept OPTIONS
     configurator.add_cors_preflight_handler()
+    # Ensure a "Cache-Control: no-store" is setup by default on all responses
+    # Avoid a bug with Firefox: https://github.com/tracim/tracim/issues/5334
+    configurator.add_subscriber(default_to_cache_control_no_store, NewResponse)
```

Comment:
This apply to error case too ? (just curious, not clear idea about caching and error case).

Reply:
Error case? AFAIK the htpp caching does not care about the http status code. And here the cache-control header will be added to all response as long as they do not already contain one.

Reply:
ok.